index.html

```diff
@@ -13,8 +13,12 @@
   </head>
   <body>
     <header>
-      <div class="logo">Logo</div>
-      <div class="content">Content</div>
+      <div class="logo">Chris</div>
+      <div class="content">
+        <a href="#hero"><li>About me</li></a>
+        <a href="#skills"><li>Skills</li></a>
+        <a href="#contact"><li>Contact</li></a>
+      </div>
```

Comment:
A semantic element would be perfect here! :)
https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i?utm_source=mybridge&utm_medium=blog&utm_campaign=read_more

```suggestion
      <nav class="content">
        <a href="#hero"><li>About me</li></a>
        <a href="#skills"><li>Skills</li></a>
        <a href="#contact"><li>Contact</li></a>
      </nav>
```